packages/react-pose/src/components/Transition/children.ts

```diff
@@ -68,11 +72,14 @@ const mergeChildren = ({
     };
 
     if (entering.has(child.key as string)) {
+      // If child is entering
       newChildProps.initialPose = preEnterPose;
-      newChildProps._pose = enterPose;
+      newChildProps._pose = enterPose; // TODO: Remove _pose and merge with child.props.pose
```

Comment:
sideways note - would it be possible to give `_pose` a better name? :p just by reading it I have no idea how it's used

Reply:
It’s basically just a private pose that I added cuz I didn’t know how to access child props